students/srepking/Lesson09/donors.py

```diff
@@ -0,0 +1,140 @@
+
+
+class Group:
+
+    def __init__(self):
+
+        self._donor_raw = {}
+
+    def get_donors(self):
```

Comment:
common practice is to use a decorator for property:
```
@property
def get_donors(self):
```

Reply:
however.. no need for decorator here if you're not applying any special logic to that attribute.

Reply:
I looked at your example showing how you made the relationship between single donor objects and the collection. Very nice! I incorporated this approach that you used, and learned from it. Thank you!

I deleted the @property for this def, but used it elsewhere.

Reply:
glad it helped!